tests/shared/models/cb-admin.test.js

```diff
@@ -0,0 +1,142 @@
+const test = require('tape');
+const pg = require('pg');
+const { getConfig } = require('../../../config');
+const { refresh: refreshDB } = require('../../../db/scripts');
+const cbAdmin = require('../../../react-backend/shared/models/cb-admin');
+
+const config = getConfig(process.env.NODE_ENV);
+
+test('Db Query | cbAdmin ', async tape => {
```

Comment:
`CbAdmin model` is a better test suite descriptor.

Reply:
I've named all my model tests like this to make it easier to read the tape output with the nested tests we're using